samples/winapi_drivelist/main.c

```diff
@@ -56,11 +65,30 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
+    goto unmount_c;
+    goto unmount_e;
+
+    debugPrint("\nDone!");
+
+    while (1) {
         Sleep(2000);
     }
 
     return 0;
-}
+
+    unmount_c:
+        ret = nxUnmountDrive('C');
+        // If there was an error while unmounting
+        if (!ret) {
+            error = GetLastError();
+            debugPrint("\nCouldn't unmount C: drive! Error code: %x\n. Trying to unmount E: drive...\n", error);
+        }
+
+    unmount_e:
+        ret = nxUnmountDrive('E');
+        if (!ret) {
+            error = GetLastError();
+            debugPrint("\nCouldn't unmount E: drive! Error code: %x\n", error);
+        }
```

Comment:
I find this indentation misleading. Indenting a block of code usually signals a scope level or that the block belongs together, but this is not the case here, it's just normal code at the same scope as the rest of the function.

I'd recommend to use the same indentation for this code as the rest of the function, but use no indentation for the label. Small example:
```c
int main(void) { 
    goto end;

end:
    return 0
}
```